server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/routes/TasksRoutes.java

```diff
@@ -133,13 +136,29 @@ public Object getStatus(Request req, Response response) {
     @ApiOperation(value = "Await, then get a task execution details")
     @ApiResponses(value = {
         @ApiResponse(code = HttpStatus.OK_200, message = "A specific class execution details", response = ExecutionDetailsDto.class),
-        @ApiResponse(code = HttpStatus.BAD_REQUEST_400, message = "The taskId is invalid"),
-        @ApiResponse(code = HttpStatus.NOT_FOUND_404, message = "The taskId is not found")
+        @ApiResponse(code = HttpStatus.BAD_REQUEST_400, message = "The taskId is invalid or invalid timeout"),
+        @ApiResponse(code = HttpStatus.NOT_FOUND_404, message = "The taskId is not found"),
+        @ApiResponse(code = HttpStatus.REQUEST_TIMEOUT_408, message = "The timeout have been reached")
```

Comment:
do you think the message add anything?

Reply:
sure, because it specifies the cause of the request timeout

Reply:
do you know other timeout cause than "The timeout have been reached"?

Reply:
The server has reached it's life timeout
The user should have reach her end of life

More seriously I'm not sure it means exactly what it should (see https://tools.ietf.org/html/rfc7231#section-6.5.7 ), but I'm fine with it anyway.

Reply:
can the result when reached timeout be represented in by 200 response code and a Json body telling that the task is in processing? and the message will be more clear than just a status code

Reply:
A 200 status code for an exceeding timeout, I disagree, we submit a request, which is the specification of a process, we can't fulfill the requirements, we should not say that it's ok.